# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/warp/bug1825220.js
# Commit: 2ce0bfc8f05b
# Full Hash: 2ce0bfc8f05bee15b935a2d71667353488f46ac9
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2023-04-05 09:36:23
# Description:
#   Bug 1825220: Don't crash on unreachable GetInlinedArgument r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D174530
# ==============================================================================

diff -r 3344d8f2cd10 -r 2ce0bfc8f05b js/src/jit-test/tests/warp/bug1825220.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/warp/bug1825220.js	Tue Apr 04 17:22:32 2023 +0000
@@ -0,0 +1,11 @@
+// |jit-test| --ion-pruning=off; --fast-warmup
+
+function foo() {
+    Date.prototype.toLocaleString()
+}
+
+for (var i = 0; i < 100; i++) {
+    try {
+	foo();
+    } catch {}
+}